starters/standard/package.json

```diff
@@ -42,7 +41,8 @@
     "@types/react-dom": "^18.3.1",
     "prisma": "^6.4.1",
     "vite": "^6.1.0",
-    "wrangler": "^4.4.0"
+    "wrangler": "^4.4.0",
+    "typescript": "5.5.4"
```

Comment:
@justinvdm We should probably remove "typescript" as a dependency in `sdk` and add it to the starters.